base_search_fuzzy/readme/ROADMAP.md

```diff
@@ -0,0 +1,4 @@
+- Modify the general search parts (e.g. in tree view or many2one fields)
+- Add better order by handling
+- This module will not be necessary from version 16 ([\[IMP\] Better
```

Comment:
@dzungtran89 I'm not sure to understand what you mean in your comment: do you mean that this module is not useful anymore given the changes upstream? if it is still useful we should probably remove this part in the ROADMAP and make explicit what is still interesting in that module

Reply:
@nilshamerlinck 
- because the upstream has already added triagram index, so we can manage these indexes at the model level
- one thing is still interesting in this module is to allow users defining triagram indexes, but then we have to manage the translation search as the upstream does [here](https://github.com/odoo/odoo/commit/ef00294e7189#diff-5f043f5617daedcdc16b95f0f4bf3f2865417f03e18b4b33f9afc86cb128599aR480)

So, my point is we don't need to this module to v17

Reply:
ok, thanks for clarifying, I was confused because you approved the PR :)

Reply:
mybad, I had approved before seeing the relevant commit of the translation part when looking deeper into the upstream